recipes/pyre/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -euo pipefail
+
+if [ `uname` = "Darwin" ]; then
+    # Use specified macOS SDK, and enforce minimum version
+    export CXXFLAGS="$CXXFLAGS -isysroot $CONDA_BUILD_SYSROOT"
+else
+    # Remove after resolving https://github.com/pyre/pyre/issues/64
+    export LDFLAGS="-lrt $LDFLAGS"
+fi
+
+mkdir build && cd build
+
+# Compute sitepackage dir relative to install prefix
+# Remove after backporting https://github.com/pyre/pyre/pull/60
+relpath(){ $PYTHON -c "import os.path; print(os.path.relpath('$1','$2'))"; }
+pypkgrel=$(relpath $SP_DIR $PREFIX)
+
+# BUILD_TESTING=y enables pyre's test suite
+# Remove after merging https://github.com/pyre/pyre/pull/60
+cmake \
+    -DBUILD_TESTING=y \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_LIBDIR=lib `# Override GNUInstallDirs - would be lib64` \
```

Comment:
the backtick is outside of the comment

Reply:
That's a little hacky trick to allow inline comments, so the subsequent lines don't turn into comments as well :P
If that's too tricky I can just move the comments elsewhere.

Reply:
Yeah, please do.